Sieve_of_Eratosthenes/SieveOfEratosthenes.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
+#include <math.h>     // for using square root 
+#include <stdbool.h>  // to include bool datatype 
+
+void sieveOfEratosthenes(int n){
+	bool numbers[n+1];
+	int i, sqrt_n = sqrt(n)+1, j;
+	
+	memset(numbers, true, sizeof(numbers));
+
```

Comment:
It's good practise. In CP, you might want to skip this, but here you creating a showcase/example code, so you have to make everything clear.